app/util/analytics.py

```diff
@@ -208,6 +223,135 @@ def generate_analytics(self):
         self.set_date_timestamp()
         self.application_version = self.get_application_version()
 
+    @property
+    def actual_git_operations_count(self):
+        count = 0
+
+        if self.application_type != BITBUCKET:
```

Comment:
Not sure why do we need this check considering the fact this function is called after additional checks on the 317, 329 rows

Reply:
extra protection in case later on someone will decide to use this property